repositories.gradle

```diff
@@ -0,0 +1,26 @@
+repositories {
+    maven {
+        name "ChickenBones Maven"
+        url "https://chickenbones.net/maven/"
+    }
+    maven {
+        name "IC2 Maven"
+        url "https://maven.ic2.player.to/"
+    }
+    ivy {
+        name "BuildCraft"
+        artifactPattern "https://www.mod-buildcraft.com/releases/BuildCraft/[revision]/[module]-[revision](-[classifier]).[ext]"
+    }
+    maven {
+        name "CurseForge Maven"
+        url "https://minecraft.curseforge.com/api/maven/"
+    }
+    ivy {
+        name = 'gtnh_download_source'
+        artifactPattern("http://downloads.gtnewhorizons.com/Mods_for_Jenkins/[module]-[revision].[ext]")
+    }
```

Comment:
all ivy repos should be moved to bottom, so they are last to be searched (it's preferred to have artifacts from a official (more stable) location like maven repos and making ivy/jitpack a last resort)